modelexpress_client/python/modelexpress/metadata/artifact_lifecycle.py

```
@@ -530,6 +535,21 @@ def triton_version() -> str:
         return ""
 
 
+def tvm_ffi_version() -> str:
+    try:
+        import tvm_ffi
+
+        version = getattr(tvm_ffi, "__version__", None)
+        if version:
+            return str(version)
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)